packages/gestalt/src/Image.js

```diff
@@ -132,6 +135,7 @@ export default class Image extends PureComponent<Props> {
         <img
           alt={alt}
           className={styles.img}
+          elementtiming={elementTiming}
```

Comment:
Might be good calling out that this will only happen when `fit` is not set to `cover` or `contain`

See `shouldScaleImage`

Reply:
updated. thanks!